lib/partition_setup.pm

```diff
@@ -40,7 +40,7 @@ our %partition_roles = qw(
 
 # We got changes to the storage-ng UI in SLE 15 SP1, Leap 15.1 and TW
 sub is_storage_ng_newui {
-    return is_storage_ng && (is_sle('15-SP1+') || is_leap('15.1+'));
+    return is_storage_ng && (is_sle('15-SP1+') || is_leap('15.1+') || is_tumbleweed && is_staging);
```

Comment:
This will break other staging projects. Can you make the detection dynamic? If not then check for :H specifically

Reply:
I will add variable to do same as we did for system roles.

Reply:
Dynamic detection is a hacky solution as it means that we don't know what we expect. Just variable won't work as we have single medium for all staging projects, so I enabled it for H specifically and that we can enable it everywhere without urgent code change. I've added this topic to the retro, the process doesn't look good to me.